octavia-cli/publish.sh

```diff
@@ -2,8 +2,11 @@
 
 set -ux
 VERSION=$1
-SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
+docker login -u airbytebot -p "${DOCKER_PASSWORD}"
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+docker run --privileged --rm tonistiigi/binfmt --install all
+docker build --push --tag airbyte/octavia-cli:${VERSION} ${SCRIPT_DIR}
 docker buildx create --name octavia_builder > /dev/null 2>&1
 set -e
```

Comment:
nit: this should be top of script.

Reply:
see example: https://github.com/airbytehq/airbyte/blob/280245322edef478bf48b4b71f73378ca451d127/airbyte-integrations/bases/base-normalization/entrypoint.sh#L36